src/cargo/util/toml/schema.rs

```diff
@@ -83,22 +83,15 @@ pub struct TomlWorkspace {
     pub metadata: Option<toml::Value>,
 
     // Properties that can be inherited by members.
```

Comment:
Is this comment still meaningful in the context? People using `TomlWorkspace` might not need to worry about workspace inheritance.

Reply:
I'm not too sure what the concern is.  From my perspective, the comment still seems relevant (these are `workspace.package`, `workspace.dependencies`, and `workspace.lints` tables).  For any of `TomlWorkspace`, the caller decides whether they care or not.